configure.ps1

```diff
@@ -48,8 +48,12 @@ Invoke-BuildStep 'Installing .NET CLI' {
 } -ev +BuildErrors
 
 # Restoring tools required for build
+.\cli\dotnet.exe msbuild .\build\config.props /v:m /nologo /t:GetAdditionalFeeds
+# Workaround: Execute dotnet.exe command twice to avoid first time startup message
+$AdditionalFeeds = .\cli\dotnet.exe msbuild .\build\config.props /v:m /nologo /t:GetAdditionalFeeds
```

Comment:
Workaround

Reply:
We can disable first time startup. 
Since we install it every time, we don't need it.

Reply:
I think the complexity of removing the one source from our nuget.config and "dynamically" adding it our non-sourcebuild builds greatly outweighs the cost of source build keeping one patch that deletes a single entry from our nuget.config.

I don't think we should do this.